projects/keycloak-angular/src/lib/core/services/keycloak.service.ts

```diff
@@ -212,7 +212,11 @@ export class KeycloakService {
     this.initServiceValues(options);
     const { config, initOptions } = options;
 
-    this._instance = Keycloak(config);
+    let initFunction = Keycloak;
+    if (typeof Keycloak == 'object') {
+      initFunction = Keycloak['default'];
+    }
+    this._instance = initFunction(config);
```

Comment:
Why this change was needed? Could you check if with version 17 all things still work as expected? 

If a major change happened in keycloak-js, we might also need a major change in the library.

Reply:
It's about keycloak-js.
Export modules (ES) and transpile.